p2p/security/noise/transport.go

```diff
@@ -65,3 +82,50 @@ func (t *Transport) WithSessionOptions(opts ...SessionOption) (sec.SecureTranspo
 	}
 	return st, nil
 }
+
+func MatchMuxers(initiatorMuxers, responderMuxers []string) string {
```

Comment:
Does this need to be exported?

Reply:
This is a private utility function that is used by the transport EDH only, so I don't see a need to export. Any other considerations?